microsetta_admin/server.py

```diff
@@ -242,7 +242,7 @@ def email_stats():
                                    resource=None,
                                    search_error=None,
                                    projects=projects)
-        emails = [email]
+        emails = [email, ]
```

Comment:
Is this stylistic or does this change functionality somehow?

Reply:
stylistic. if using a different iterable, like a `tuple`, it would be a behavior difference

Reply:
Okay, as long as it doesn't turn the string email into the tuple (email,) and then construct the list of the tuple [(email,)] I'm happy 👍

Reply:
nope :) I've begun to lean toward being defensive here out of habit because of this:

```python
>>> ['foo']
['foo']
>>> ['foo', ]
['foo']
>>> ('foo')
'foo'
>>> ('foo', )
('foo',)
```